env/staging/create_metrics_lambda/lambda/create_metric.test.js

```diff
@@ -74,5 +76,50 @@ describe("handler", () => {
     })
 
     delete process.env.TABLE_NAME
+    delete process.env.SPLIT_THRESHOLD
+  })
+
+  it("saves large event body successfully after splitting", async () => {
```

Comment:
Maybe add one more test for the case where it splits into 2 chunks (where each chunk has multiple metrics) to make sure we only write the 2 chunks and not the individual metrics.

Reply:
good idea. i'll add that!